### DISCUSSION TITLE
m1  ARM64  Version Support!!!!!!!!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by wenzhongfa** [pyg-team/pytorch_geometric#3540] [Q&A]
Please Support!!!!!!!    I really hope!!!!!!!!   Thanks!  Quikly!!!

**Comment by rusty1s** [comment]
While you cannot use the pre-built packages for M1 macs yet (we are waiting on GitHub actions to support them), it should be possible to install from source:
```
pip install --verbose torch-scatter
```
If you encounter any issues, please send the installation logs.
